<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPlainTextEdit" name="hex_view">
         <property name="minimumSize">
          <size>
           <width>450</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Courier New</family>
          </font>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="Info">
         <property name="minimumSize">
          <size>
           <width>220</width>
           <height>0</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Header</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QWidget" name="widget_2" native="true">
               <layout class="QFormLayout" name="formLayout">
                <property name="horizontalSpacing">
                 <number>0</number>
                </property>
                <property name="verticalSpacing">
                 <number>0</number>
                </property>
                <property name="margin">
                 <number>0</number>
                </property>
                <item row="0" column="0">
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>preamnble</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>version</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>checksum</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>msg_size</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>msg_num</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="0">
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>write_errors</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="header_preamble"/>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="header_version"/>
                </item>
                <item row="2" column="1">
                 <widget class="QLineEdit" name="header_checksum"/>
                </item>
                <item row="3" column="1">
                 <widget class="QLineEdit" name="header_msg_size"/>
                </item>
                <item row="4" column="1">
                 <widget class="QLineEdit" name="header_msg_num"/>
                </item>
                <item row="5" column="1">
                 <widget class="QLineEdit" name="header_write_errors"/>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>LTC</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLineEdit" name="ltc_ltc"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>RTC</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLineEdit" name="rtc_rtc"/>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_4">
               <property name="title">
                <string>NAV_SOL</string>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="margin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QWidget" name="widget_3" native="true">
                  <layout class="QFormLayout" name="formLayout_2">
                   <property name="horizontalSpacing">
                    <number>0</number>
                   </property>
                   <property name="verticalSpacing">
                    <number>0</number>
                   </property>
                   <property name="margin">
                    <number>0</number>
                   </property>
                   <item row="1" column="0">
                    <widget class="QLabel" name="label_7">
                     <property name="text">
                      <string>ITOW</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="1">
                    <widget class="QLineEdit" name="nav_sol_itow"/>
                   </item>
                   <item row="2" column="0">
                    <widget class="QLabel" name="label_8">
                     <property name="text">
                      <string>FTOW</string>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="1">
                    <widget class="QLineEdit" name="nav_sol_ftow"/>
                   </item>
                   <item row="3" column="0">
                    <widget class="QLabel" name="label_9">
                     <property name="text">
                      <string>week</string>
                     </property>
                    </widget>
                   </item>
                   <item row="4" column="0">
                    <widget class="QLabel" name="label_10">
                     <property name="text">
                      <string>gpsFix</string>
                     </property>
                    </widget>
                   </item>
                   <item row="5" column="0">
                    <widget class="QLabel" name="label_11">
                     <property name="text">
                      <string>Flags</string>
                     </property>
                    </widget>
                   </item>
                   <item row="6" column="0">
                    <widget class="QLabel" name="label_12">
                     <property name="text">
                      <string>ECEF</string>
                     </property>
                    </widget>
                   </item>
                   <item row="7" column="0">
                    <widget class="QLabel" name="label_13">
                     <property name="text">
                      <string>ECEFV</string>
                     </property>
                    </widget>
                   </item>
                   <item row="8" column="0">
                    <widget class="QLabel" name="label_14">
                     <property name="text">
                      <string>pAcc/sAcc</string>
                     </property>
                    </widget>
                   </item>
                   <item row="9" column="0">
                    <widget class="QLabel" name="label_15">
                     <property name="text">
                      <string>pdop/numSv</string>
                     </property>
                    </widget>
                   </item>
                   <item row="3" column="1">
                    <widget class="QLineEdit" name="nav_sol_week"/>
                   </item>
                   <item row="4" column="1">
                    <widget class="QLineEdit" name="nav_sol_gpsfix"/>
                   </item>
                   <item row="5" column="1">
                    <widget class="QLineEdit" name="nav_sol_flags"/>
                   </item>
                   <item row="6" column="1">
                    <widget class="QLineEdit" name="nav_sol_ecef"/>
                   </item>
                   <item row="7" column="1">
                    <widget class="QLineEdit" name="nav_sol_ecefv"/>
                   </item>
                   <item row="8" column="1">
                    <widget class="QLineEdit" name="nav_sol_pacc_sacc"/>
                   </item>
                   <item row="9" column="1">
                    <widget class="QLineEdit" name="nav_sol_pdop_numsv"/>
                   </item>
                   <item row="0" column="1">
                    <widget class="QLineEdit" name="nav_sol_pps"/>
                   </item>
                   <item row="0" column="0">
                    <widget class="QLabel" name="label_16">
                     <property name="text">
                      <string>PPS</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QSlider" name="rangeSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="rangeSpinBox">
         <property name="maximum">
          <number>999999999</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="packetCountEdit">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuNavigate">
    <property name="title">
     <string>Navigate</string>
    </property>
    <addaction name="actionNext"/>
    <addaction name="actionPrev"/>
   </widget>
   <addaction name="menuOpen"/>
   <addaction name="menuNavigate"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="text">
    <string>Next</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="actionPrev">
   <property name="text">
    <string>Prev</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>rangeSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>rangeSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>483</x>
     <y>548</y>
    </hint>
    <hint type="destinationlabel">
     <x>724</x>
     <y>560</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rangeSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>rangeSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>724</x>
     <y>560</y>
    </hint>
    <hint type="destinationlabel">
     <x>643</x>
     <y>559</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>groupBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>widget_2</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>578</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>629</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>groupBox_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>ltc_ltc</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>780</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>780</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>groupBox_3</sender>
   <signal>clicked(bool)</signal>
   <receiver>rtc_rtc</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>780</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>780</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>groupBox_4</sender>
   <signal>clicked(bool)</signal>
   <receiver>widget_3</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>593</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>630</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
